CHANGELOG.md

```diff
@@ -1,61 +1,12 @@
 # Change log
 
-## 2.0.0
```

Comment:
expand the changelog, do not replace it

--------

 - 2.0.1 
   - breaking changes
   - new features
   - fixed bugs
 - 2.0.0 
   - breaking changes
   - new features
   - fixed bugs
 
etc...